wordpress-mobile/WordPress-Android
# Add initial Quick Start card bottom sheet menu implementation

Fixes #13583

This is a follow-up to #13724, so make sure check that one out first.

This PR introduces a basic bottom sheet implementation for when the Quick Start card overflow button is clicked. It contains three options:

- Pin
- Hide
- Don't show again

The logic for clicking any of those options should be implemented in future PRs, so for now we're only interested to correctly display the bottom sheet and to correctly map each click to its correct Quick Start card:

https://user-images.githubusercontent.com/830056/104062403-2ecef400-51d9-11eb-930c-8841ba217f02.mp4

### To test

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen.
1. Notice the 2 sample Quick Start cards.
1. Tap on the overflow button in the top right corner of either of the cards.
1. Notice the bottom sheet menu.
1. Tap any of the options.
1. Notice a toast message appears with the correct information about the Quick Start card and option clicked.

---

PR submission checklist:
